### DISCUSSION TITLE
Vendor's Release Notes for the 3.7.0

### DISCUSSION THREAD

**Discussion by zafersn** [zephyrproject-rtos/zephyr#75317] [General] [answered]
I was wondering if we have any vendor release notes / histories for zephyr 3.7.0 especially, I m looking for st/stm32 board chances for 3.7.0.

I had a quick glance at https://docs.zephyrproject.org/latest/releases/release-notes-3.7.html 
But I didnt see anything for overall st implementation. I know some changes I need, for example, st included support for gpio pin as a wake-up source, but what else?

**Comment by erwango** [comment]
This is on going.

**Reply by erwango** [reply, marked as answer]
FYI https://github.com/zephyrproject-rtos/zephyr/pull/75325

**Reply by zafersn** [reply]
> FYI #75325
thank you! @erwango
